src/components/listitems/ListItemTransaction.tsx

```diff
@@ -200,19 +159,29 @@ export const ListItemTransaction = ({
             {subtitle}
           </LabelSmall>
         </View>
-        <View style={Styles.rightSection}>
-          <TransactionAmountOrBadgeComponent />
-          {hasChevronRight && (
-            <Icon
-              name="chevronRightListItem"
-              color={interactiveColor}
-              size={IOListItemVisualParams.chevronSize}
-            />
-          )}
-        </View>
-      </>
-    );
-  };
+      </HStack>
+      <HStack style={{ alignItems: "center" }}>
+        {hideAmount ? (
```

Comment:
Instead of using this additional prop, what do you think about hiding the amount when a badge is provided as input?
```suggestion
        {badge ? (
```

Reply:
@Hantex9 The refactoring was done incrementally, so I completely ignored this optimisation. Good catch, fixed with commit https://github.com/pagopa/io-app-design-system/pull/353/commits/f58901d45dfd428e85bf534652319fcf713e8643)